infra/ibmcloud/terraform/k8s-s390x-infra-setup/outputs.tf

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+output "secrets_manager_id" {
```

Comment:
this is not a module so shouldn't required `variables.tf` or `outputs.tf`

Reply:
don't copy paste across statefiles though - either refer to the state itself or use the AWS provider to refer to the value directly